docs/internals/howto-release-django.txt

```diff
@@ -418,7 +498,7 @@ Now you're ready to actually put the release out there. To do this:
 
    .. code-block:: shell
 
-       $ twine upload -s dist/*
+       $ twine upload dist/*
```

Comment:
PyPI no longer supports uploading signatures. If uploaded, the following email is received:

```
[PyPI] Unsupported GPG signature uploaded to PyPI

What?
During your recent upload of Django to PyPI, we noticed you uploaded a GPG signature. However, support for GPG signatures has been removed from PyPI.

What should I do?
While uploads will continue to work, any signatures uploaded to PyPI will be discarded. It is recommended to no longer upload signatures to PyPI.
```